mysys/system_key.c

```diff
@@ -0,0 +1,59 @@
+#include "system_key.h"
```

Comment:
Some sort of Percona lawyer thing comment

Reply:
Done

Reply:
Verbose one please